src/main/java/org/cloudfoundry/promregator/endpoint/MetricsEndpoint.java

```diff
@@ -29,6 +30,9 @@
 public class MetricsEndpoint extends AbstractMetricsEndpoint {
 	private static final Logger log = Logger.getLogger(MetricsEndpoint.class);
 
+	@Value("${promregator.single.endpoint.max:-1}")
```

Comment:
If we introduce a new config option, then it also has to be explained in https://github.com/promregator/promregator/blob/3330cd742f3710b89a855cdf80c47f2e2f116f0c/docs/config.md